frontend/simple/views/CreateGroup.vue

```diff
@@ -1,32 +1,135 @@
+                </div>
+                <div class="tile notification">
+                  <p class="title"><i18n>What percentage of members are required to change the rules?</i18n></p>
+                  <p class="title percentage">{{changePercentage}}%</p>
+                  <input type="range" v-validate data-vv-as="Percentage to change rules" data-vv-rules="between:1,100"  min="0" max="100" name="changePercentage" v-model="changePercentage">
+                  <span v-show="errors.has('changePercentage')" class="help is-danger">The Percentage to change rules field must be between 1% and 100%</span>
+                  <input v-if="dev" type="text" v-model="changePercentage" name="proxyChangePercentage" style="opacity:0; position: absolute;">
+                </div>
+            </div>
+            <div class="tile is-4 is-vertical question-group">
+              <div class="tile notification is-warning">
+                <p class="title"><i18n>Is your group open to new members?</i18n></p>
+                <p class="title" v-show="openMembership">Yes</p>
+                <p class="title" v-show="!openMembership">No</p>
+                <label class="switch">
+                  <input type="checkbox" name="openMembership" v-model="openMembership">
+                  <div class="slider round"></div>
+                </label>
+              </div>
+              <div class="tile notification">
+                <p class="title"><i18n>How many members should it take to approve a new member?</i18n></p>
+                <p class="title">{{memberApprovalPercentage}}%</p>
+                <input type="range" min="0" max="100" data-vv-as="Member Approval Percentage" v-validate data-vv-rules="between:1,100" name="memberApprovalPercentage" v-model="memberApprovalPercentage">
+                <span v-show="errors.has('memberApprovalPercentage')" class="help is-danger">The Member Approval Percentage field must be between 1% and 100%.</span>
+                <input v-if="dev" type="text" v-model="memberApprovalPercentage" name="proxyMemberApprovalPercentage" style="opacity:0; position: absolute;">
+              </div>
+              <div class="tile notification">
+                <p class="title"><i18n>How many members should it take to remove a member?</i18n></p>
+                <p class="title">{{memberRemovalPercentage}}%</p>
+                <input type="range" min="0" max="100" data-vv-as="Member Removal Percentage" v-validate data-vv-rules="between:1,100" name="memberRemovalPercentage" v-model="memberRemovalPercentage">
+                <span v-show="errors.has('memberRemovalPercentage')" class="help is-danger">The Member Removal Percentage field must be between 1% and 100%.</span>
+                <input v-if="dev" type="text" v-model="memberRemovalPercentage" name="proxyMemberRemovalPercentage" style="opacity:0; position: absolute;">
```

Comment:
Can you add a comment here and also in the frontend test to explain this weirdness of having two things bound to `memberRemovalPercentage`?